src/Azure/Orleans.Transactions.AzureStorage/Storage/AzureTransactionLogStorage.cs

```diff
@@ -322,6 +357,30 @@ private List<CommitRecord> DeserializeCommitRecords(byte[] serializerCommitRecor
             return commitRecords;
         }
 
+        public class ArchivalRow : TableEntity
+        {
+            public ArchivalRow()
+            {
+            }
+
+            public ArchivalRow(byte[] transactions, long firstTransactionId, long firstLSN, string clusterId, DateTime archivalTime)
+            {
+                this.Transactions = transactions;
+                this.ArchivalTime = archivalTime;
```

Comment:
we shouldn't need archive time as a property, as the entity should already have a timestamp on it.

Reply:
ok